cmake/Conan.cmake

```diff
@@ -18,7 +18,7 @@ conan_cmake_run(
   ${CONAN_EXTRA_REQUIRES}
   catch2/2.11.0
   docopt.cpp/0.6.2
-  fmt/6.0.0
+  fmt/6.1.2
```

Comment:
This looks like a strange thing to change, but the Mac build consistently fails when fmt is set to 6.0.0. It also fails on my Fedora box. Updating fmt to 6.1.2 appears to correct the issue.

See this issue for reference: https://github.com/lefticus/cpp_weekly_game_project/issues/3